crates/router/src/types/api/admin.rs

```diff
@@ -1,11 +1,40 @@
 pub use api_models::admin::{
     CreateMerchantAccount, CustomRoutingRules, DeleteMcaResponse, DeleteResponse,
-    MerchantConnectorId, MerchantDetails, MerchantId, PaymentConnectorCreate, PaymentMethods,
-    WebhookDetails,
+    MerchantAccountResponse, MerchantConnectorId, MerchantDetails, MerchantId,
+    PaymentConnectorCreate, PaymentMethods, WebhookDetails,
 };
 
+use crate::types::{
+    storage,
+    transformers::{Foreign, ForeignInto},
+};
+
+impl From<Foreign<storage::MerchantAccount>> for Foreign<MerchantAccountResponse> {
+    fn from(value: Foreign<storage::MerchantAccount>) -> Self {
+        let item = value.0;
+        MerchantAccountResponse {
+            merchant_id: item.merchant_id,
+            merchant_name: item.merchant_name,
+            api_key: item.api_key,
+            return_url: item.return_url,
+            enable_payment_response_hash: item.enable_payment_response_hash,
+            payment_response_hash_key: item.payment_response_hash_key,
+            redirect_to_merchant_with_http_post: item.redirect_to_merchant_with_http_post,
+            merchant_details: item.merchant_details,
+            webhook_details: item.webhook_details,
+            routing_algorithm: item.routing_algorithm.map(ForeignInto::foreign_into),
+            custom_routing_rules: item.custom_routing_rules,
+            sub_merchants_enabled: item.sub_merchants_enabled,
+            parent_merchant_id: item.parent_merchant_id,
+            publishable_key: item.publishable_key,
+            metadata: item.metadata,
+            locker_id: item.locker_id,
+        }
+        .into()
+    }
+}
```

Comment:
Move this to `types/transformers.rs`.

Reply:
Such implementations are placed in types/api , I referred from here. https://github.com/juspay/hyperswitch/blob/e7579a4819358001768d2c6a5b04e10f8810e24f/crates/router/src/types/api/payments.rs#L114